### PR TITLE
Meter madness

### PR DESCRIPTION
**Pull request by robojumper**

### CODE REVIEW

File: include/toBeSorted/scgame.h [hunk 1]
```diff
@@ -2,9 +2,10 @@
 #define SCGAME_H
 
 #include "common.h"
+#include "sized_string.h"
 
 struct SpawnInfo {
-    /* 0x00 */ char stageName[32]; // Probably SizedString<32>
+    /* 0x00 */ SizedString<32> stageName; // Probably SizedString"32"
```
Comment (elijah-thomas774) [inline include/toBeSorted/scgame.h:8]: comment can be removed I think xD

File: include/s/s_State.hpp [hunk 2]
```diff
@@ -25,5 +25,12 @@
...
 
-#define UI_STATE_MGR_DECLARE(class_name) sFStateMgr_c<class_name, sStateMethodUsr_FI_c> mStateMgr;
+#define UI_STATE_MGR_DECLARE(class_name) \
+    sFStateMgr_c<class_name, sStateMethodUsr_FI_c> mStateMgr;
+
+#define STATE_MGR(class_name) \
+    sFStateMgr_c<class_name, sStateMethodUsr_FI_c>
+
+#define UI_STATE_MGR_DECLARE(class_name) \
+    sFStateMgr_c<class_name, sStateMethodUsr_FI_c> mStateMgr;
```
Comment (elijah-thomas774) [inline include/s/s_State.hpp:35]: `UT_STATE_MGR_DECLARE` is duplicated

Comment (robojumper) [reply include/s/s_State.hpp:35]:
```suggestion
#define UI_STATE_MGR_DECLARE(class_name) \
    sFStateMgr_c<class_name, sStateMethodUsr_FI_c> mStateMgr;

#define STATE_MGR(class_name) \
    sFStateMgr_c<class_name, sStateMethodUsr_FI_c>

```

File: include/toBeSorted/scgame.h (hunk 1 above)
Comment (robojumper) [reply include/toBeSorted/scgame.h:8]:
```suggestion
    /* 0x00 */ SizedString<32> stageName;
```

**Pull request merged**